content/docs/concepts/security/index.md

```diff
@@ -461,10 +460,24 @@ peers:
   - mtls: {}
 {{< /text >}}
 
-Currently, the mutual TLS setting doesn't require any parameters. Hence,
-`-mtls: {}`, `- mtls:` or `- mtls: null` declarations are treated the same. In
-the future, the mutual TLS setting may carry arguments to provide different
-mutual TLS implementations.
+The mutual TLS setting has an optional `mode` parameter that defines the
+strictness of the peer transport authentication:
+
+- `mode: STRICT` enforces that all peers must use mutual TLS. This declaration
+is identical to `-mtls: {}`, `- mtls:` or `- mtls: null`
+
+- `mode: PERMISSIVE` allows peers to use mututal TLS while still allowing
+peers to not use transport authentication. This mode is useful for transitioning
+existing workloads to peer authentication gradually, as it allows interoperation
+between mTLS and plaintext traffic.
+
+If this section is omitted completely, peers may not use transport
+authentication, and mutual TLS connections bound for the sidecar pod
+will be rejected. However clients are free to handle their own mutual
```

Comment:
Perhaps this wording is confusing. Happy to remove the part about clients handling mTLS, but I thought it might be prudent to hint that this is an option, and to make sure users know that the 'HTTP' language provided by istioctl isn't really HTTP, it's more just 'plaintext' / 'auth how you like'

Reply:
The paragraph reads a bit awkwardly, consider:
When you do not specify a mutual TLS mode, peers can't use transport
authentication, and Istio rejects mutual TLS connections bound for the sidecar. 
At the application layer, services can still handle their own mutual TLS sessions.